docs/reference/sql/language/data-types.asciidoc

```diff
@@ -33,6 +36,7 @@ s|SQL precision
 | <<text, `text`>>            | text          | VARCHAR     | 2,147,483,647
 | <<binary, `binary`>>        | binary        | VARBINARY   | 2,147,483,647
 | <<date, `date`>>            | datetime      | TIMESTAMP   | 24
+|                             | date          | DATE        | 24
```

Comment:
This should be in the interval table below which contains the SQL -specific types not available in ES.